src/encryptsave/mod.rs

```diff
@@ -0,0 +1,175 @@
+mod ll;
+pub mod errors;
+
+
+macro_rules! tox_res {
+    ( $res:ident <- $rexpr:expr, $err:ident, $r:expr ) => {
```

Comment:
`res` is` result` abbreviation, `ident` like https://github.com/suhr/rstox/blob/master/src/core/mod.rs#L348

Maybe I should be changed to `out: indent`?